src/onnx/parse_constant.cpp

```diff
@@ -39,16 +39,41 @@ struct parse_constant : op_parser<parse_constant>
                           onnx_parser::node_info info,
                           const std::vector<instruction_ref>& /*args*/) const
     {
-        literal v = parser.parse_value(info.attributes.at("value"));
+        static const std::vector<std::string> attributes = {
+            "value", "value_float", "value_floats", "value_int", "value_ints"};
+
+        const auto p  = [&](const std::string& a) { return contains(info.attributes, a); };
+        const auto it = std::find_if(attributes.begin(), attributes.end(), p);
+
+        if(it == attributes.end())
+        {
+            MIGRAPHX_THROW("Constant node does not contain any supported attribute");
+        }
+
+        if(std::any_of(std::next(it), attributes.end(), p))
+        {
+            std::string err = "Constant contains multiple attributes: " + *it;
+            for(auto i = std::next(it); i != attributes.end(); ++i)
+            {
+                if(p(*i))
+                {
+                    err += ", " + *i;
+                }
+            }
+            MIGRAPHX_THROW(err);
+        }
```

Comment:
Use `copy_if` to copy all the attribute names to a vector and then if the vector > 1 throw an error about multiple attributes. You can use `join_strings` to convert the vector of strings to a comma-delimited string.